### PR TITLE
Create 2023-03-31-content.md

### PR DESCRIPTION
**Pull request by ptobia**
## Purpose of this pull request

This pull request (PR) updates CSE content release notes for March 31, 2023

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-cse/2023-03-31-content.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: March 31, 2023 - Content Release
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Within this content release we have deprecated two of our First Seen rules linked to low fidelity as we continue to perform internal testing around similar detections. Additionally, we are bringing out a new set of Carbon Black mappers, expanding on our existing normalization with the product.
```
Comment (kimsauce) [inline blog-cse/2023-03-31-content.md:10]:
```suggestion
Within this content release, we have deprecated two of our First Seen rules linked to low fidelity as we continue to perform internal testing around similar detections. Additionally, we are bringing out a new set of Carbon Black mappers, expanding on our existing normalization with the product.
```

File: blog-cse/2023-03-31-content.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+#### Rules
+[Deleted] FIRST-S00011 First Seen Sysmon IMPHASH - Global
+[Deleted] FIRST-S00012 First Seen Sysmon IMPHASH - Host
```
Comment (kimsauce) [inline blog-cse/2023-03-31-content.md:14]: @ptobia , I think these (and the other sections) are supposed to be bulleted

**Review by kimsauce — CHANGES_REQUESTED** [review]
style edits

File: blog-cse/2023-03-31-content.md (hunk 2 above)
Comment (jpipkin1) [reply blog-cse/2023-03-31-content.md:14]: Agreed.

**Review by jpipkin1 — COMMENTED** [review]
@ptobia, after you make the changes @kimsauce suggests (mainly to add bullets), you ought to be good to go.

File: blog-cse/2023-03-31-content.md (hunk 2 above)
Comment (ptobia) [reply blog-cse/2023-03-31-content.md:14]: will fix.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**